pyrate/gdal_python.py

```diff
@@ -274,8 +300,20 @@ def crop_resample_average(
     tmp_ds = gdal.GetDriverByName('MEM').CreateCopy('', dst_ds) \
         if (match_pyrate and new_res[0]) else None
 
+    src_ds, src_ds_mem = _setup_source(input_tif)   
+
+    if coh_path and coh_thresh:
+        coh_raster = prepifg.dem_or_ifg(coh_path)
```

Comment:
Its a general observation; when I started reading the code "coh" confused me, now that i know its abbreavation of Coherence its a breeze. I think it will be a good idea to expand it coh to coherence in var names.

Reply:
Fair point, I've expanded 'coh_' to 'coherence_'